molpx/tests/test_nbtools.py

```diff
@@ -0,0 +1,44 @@
+__author__ = 'gph82'
+
+import unittest
+import os
+import numpy as np
+import molpx
+
+class TestNbtools(unittest.TestCase):
+
+    def test_example_notebooks(self):
+        molpx.example_notebooks(dry_run=False)
+        molpx.example_notebooks(dry_run=False, extra_flags_as_one_string="--no-browser")
+
```

Comment:
@marscher , any way I can wrap lines 11 and 12 in something that will return an exception that I can handle gracefully? I'm trying some combination of timer (from threading) and sys.exit(), but no joy..
```python
def exitfunc():
    print("Exiting after idling")
    sys.exit(0)
    
def test_example_notebooks(self):
        try:
            Timer(10, exitfunc).start() # stop after 10 s
            molpx.example_notebooks(dry_run=False)
        except SystemExit:
            pass
```